docs/includes/net-standard-table.md

```diff
@@ -0,0 +1,20 @@
+| .NET Standard              | [1.0] | [1.1]  | [1.2] | [1.3] | [1.4] | [1.5] | [1.6] | [2.0] |
+|----------------------------|-------|--------|-------|-------|-------|-------|-------|-------|
+| .NET Core                  | 1.0   | 1.0    | 1.0   | 1.0   | 1.0   | 1.0   | 1.0   | 2.0   |
+| .NET Framework             | 4.5   | 4.5    | 4.5.1 | 4.6   | 4.6.1 | 4.6.1 | 4.6.1 | 4.6.1 |
```

Comment:
I personally think we can make this public document have both the current and the future expected information. It will give folks a heads up that there are some changes coming and something to think about, so I would put it both here and in the standard repo.